crypto/cmp/cmp_status.c

```diff
@@ -180,47 +161,48 @@ int ossl_cmp_pkisi_pkifailureinfo_check(const OSSL_CMP_PKISI *si, int bit_index)
     return ASN1_BIT_STRING_get_bit(si->failInfo, bit_index);
 }
 
-/*
+/*-
  * place human-readable error string created from PKIStatusInfo in given buffer
  * returns pointer to the same buffer containing the string, or NULL on error
  */
-char *OSSL_CMP_CTX_snprint_PKIStatus(OSSL_CMP_CTX *ctx, char *buf,
-                                     size_t bufsize)
+static
+char *snprint_PKIStatusInfo_parts(long status, long fail_info,
```

Comment:
Why do we have "long" here? We try to avoid that type

Reply:
Oops, these two were just neglected when cleaning up `long` -> `int` somewhat earlier. Fixed.